Automattic/wp-calypso
# Add/trial my plan page

#### Proposed Changes

* This PR adds the custom My Plan page for the eCommerce trial Plan.
* It's still missing the final copy and actions for the buttons/links.

#### Testing Instructions
* Open the "My Plan" page (`/plans/my-plan/<site-slug>`) for a site with the eCommerce trial plan
* Check that the new design is shown and that it matches the Figma design (jBwlERMS350NqhJNHZzAVF-fi-1%3A69465)

Figma:
![tyYgc4.png](https://user-images.githubusercontent.com/3801502/215928979-8175de74-a5ee-44d4-92b8-e0e3355b1fc0.png)

HTML:
![U72zie.png](https://user-images.githubusercontent.com/3801502/215929271-8cc33a0f-1e3e-4218-9b24-4086b9a4f3ba.png)

* Switch to mobile view and compare again:

Figma: 
![image](https://user-images.githubusercontent.com/3801502/216210028-b2f88ade-90a3-4e55-a8e2-132908148377.png)

HTML:
![I0uIq9.png](https://user-images.githubusercontent.com/3801502/216216901-5d57a845-c240-4ead-8ace-db3a858d5896.png)

* On mobile view, only 4 "Included features" should be displayed initially, with the option to "View all" when clicking the button.
![image](https://user-images.githubusercontent.com/3801502/216491127-b8e8001f-82d4-48a4-9f56-11ede0ee2eee.png)



Closes https://github.com/Automattic/wp-calypso/issues/71808
